src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -206,6 +210,82 @@ public WindowState WindowState
             }
         }
 
+        public WindowTransparencyLevel TransparencyLevelHint
+        {
+            get => _transparencyLevel;
+            set
+            {
+                var oldValue = _transparencyLevel;
+                _transparencyLevel = EnableBlur(value);
+
+                if (oldValue != _transparencyLevel)
+                {
+                    TransparencyLevelChanged?.Invoke(_transparencyLevel);
+                }
+            }
+        }
+
+        private WindowTransparencyLevel EnableBlur(WindowTransparencyLevel transparencyLevel)
+        {
+            bool canUseTransparency = false;
+            bool canUseAcrylic = false;
+
+            if (Win32Platform.WindowsVersion.Major >= 10)
+            {
+                canUseTransparency = true;
+
+                if (Win32Platform.WindowsVersion.Build >= 19628)
```

Comment:
This is a bit silly request but this won’t work on Windows 11 until they reach 19628 build number :)

Reply:
fixed.